business_rules/utils.py

```diff
@@ -85,3 +86,86 @@ def params_dict_to_list(params):
             'field_type': param_field_type
         } for name, param_field_type in params.items()
     ]
+
+
+def validate_rule_data(variables, actions, rule):
+    """
+    validate_rule_data is used to check a generated rule against a set of variables and actions
+    :param variables:
+    :param actions:
+    :param rule:
+    :return: bool
+    :raises AssertionError:
+    """
+    def validate_root_keys(rule):
+        """
+        Check the root object contains both 'actions' & 'conditions'
+        """
+        root_keys = rule.keys()
+        if 'actions' not in root_keys:
+            raise AssertionError('Missing "{}" key'.format('actions'))
+        if 'conditions' not in root_keys:
+            raise AssertionError('Missing "{}" key'.format('conditions'))
+
+    def validate_condition_operator(condition, rule_schema):
+        """
+        Check provided condition contains a valid operator
+        """
+        for item in rule_schema.get('variables'):
+            if item.get('name') == condition.get('name'):
```

Comment:
it seems correct, but there is a dependency on the execution order of the two methods... 
Can you add a return value despite for the current implementation that should never happen?